runtime/zephir.c

```diff
@@ -108,7 +108,7 @@ static void zephir_initialize_memory(zend_zephir_globals *zephir_globals_ptr TSR
 static void zephir_compile_program(zval *program TSRMLS_DC)
 {
 	HashTable           *ht = Z_ARRVAL_P(program);
-	HashPosition        pos = {0};
+	HashPosition        pos;
```

Comment:
@sjinks Why it was = {0} by default? Maybe you will give a cool explanation, because I am rly interested in this